libs/core/src/lib/calendar/calendar-views/calendar-day-view/calendar-day-view.component.ts

```diff
@@ -212,20 +212,22 @@ export class CalendarDayViewComponent implements OnInit, OnChanges, OnDestroy {
 
     /** @hidden
      *  Amount of selected days
-     *  0, when none,
-     *  1, when only startDate, or endDate same as startDate,
-     *  2, when both
+     *  0, when none of selected, or start date is invalid,
```

Comment:
My English is not good, but doesn't this mean exactly the same as point 2? (both are valid)
@stefanoScalzo some help here?

Reply:
should be
0, when selected and startDate are valid
1, when given valid startDate and endDate is same as startDate

Reply:
It has been changed, please take a look on it now.